docs/installation-and-operations/changing-to-bim-edition/README.md

```diff
@@ -77,3 +77,33 @@ OpenProject installations with a valid Enterprise Edition token can switch to th
 theme.
 
 Go to _-> Administration -> Design_ and from the _Themes_ drop down menu choose _OpenProject BIM_.
+
+### Docker
+
+##### OpenProject BIM Edition with Docker Compose
+
+If you would like to use OpenProject BIM Edition inside your docker-compose containers, in your `docker-compose.yml` file in `x-op-app` > `environment` add the line
```

Comment:
I have doubts that such a concept of a "docker-compose container" exists. There are docker containers.

Reply:
true its not a container, its an docker-compose installation
If you would like to use BIM Edition inside your docker-compose  OpenProject installation.

Reply:
done